xml/Microsoft.Extensions.DependencyInjection/DefaultServiceProviderFactory.xml

```diff
@@ -25,6 +25,12 @@
       <InterfaceName>Microsoft.Extensions.DependencyInjection.IServiceProviderFactory&lt;Microsoft.Extensions.DependencyInjection.IServiceCollection&gt;</InterfaceName>
     </Interface>
   </Interfaces>
+  <Attributes>
+    <Attribute FrameworkAlternate="dotnet-plat-ext-7.0">
+      <AttributeName Language="C#">[System.Diagnostics.CodeAnalysis.RequiresDynamicCode("Using Microsoft.Extensions.DependencyInjection requires generating code dynamically at runtime. For example, when using enumerable and generic ValueType services.")]</AttributeName>
+      <AttributeName Language="F#">[&lt;System.Diagnostics.CodeAnalysis.RequiresDynamicCode("Using Microsoft.Extensions.DependencyInjection requires generating code dynamically at runtime. For example, when using enumerable and generic ValueType services.")&gt;]</AttributeName>
+    </Attribute>
+  </Attributes>
   <Docs>
     <summary>Default implementation of <see cref="T:Microsoft.Extensions.DependencyInjection.IServiceProviderFactory`1" />.</summary>
     <remarks>To be added.</remarks>
```

Comment:
Line 165 of this file. Could the suggestion be fixed by forcing the inheritdoc to point to the specific cref of the interface this API implements?:

```
        <inheritdoc cref="M:Microsoft.Extensions.DependencyInjection.IServiceProviderFactory`1.CreateServiceProvider(`0)" />
```

Reply:
Worth a try - or we could just delete the `<inheritdoc />` line.